### PR TITLE
add roads transportation page

### PR DESCRIPTION
**Pull request by gregbunce**
a few of the blog links do not work b/c we're missing those blog pages. we could route those to the sgid update  landing page (/blog/categories/sgid-updates)

### CODE REVIEW

**Comment by gregbunce** [comment]
fyi: i'm effectively out of the office until next monday (18th).

File: src/data/downloadMetadata.ts [hunk 1]
```diff
@@ -2035,9 +2035,9 @@ export const dataPages: DownloadMetadata = {
     layerId: 0,
   },
   'Utah Roads': {
-    itemId: '478fbef62481427f95a3510a4707b24a',
+    itemId: '99lidPhWCzftIe9K',
```
Comment (steveoh) [inline src/data/downloadMetadata.ts:2038]: **issue**: itemid should never change

File: src/data/downloadMetadata.ts [hunk 2]
```diff
@@ -2035,9 +2035,9 @@ export const dataPages: DownloadMetadata = {
...
     name: 'Utah Roads',
-    featureServiceId: null,
+    featureServiceId: 'UtahRoads',
     openSgid: '',
```
Comment (steveoh) [inline src/data/downloadMetadata.ts:2041]: **issue**: wheres the open sgid layer name

File: src/data/downloadMetadata.ts [hunk 3]
```diff
@@ -2035,9 +2035,9 @@ export const dataPages: DownloadMetadata = {
     layerId: 0,
   },
   'Utah Roads': {
-    itemId: '478fbef62481427f95a3510a4707b24a',
+    itemId: '99lidPhWCzftIe9K',
     name: 'Utah Roads',
-    featureServiceId: null,
+    featureServiceId: 'UtahRoads',
```
Comment (steveoh) [inline src/data/downloadMetadata.ts:2040]: **issue**: wheres the open sgid layer name

**Comment by steveoh** [comment]
> a few of the blog links do not work b/c we're missing those blog pages. 

every blog post is still published just not shown. Use the content/blog folder structure to see them.

File: src/pages/products/sgid/transportation/road-centerlines.astro [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+      with questions about this data layer.
+    </p>
+  </Fragment>
+
+  <UpdateHistory slot="update-history" updates={page.updateHistory} style="secondary" />
+
+  <Section title="More resources" slot="more-resources" titlePosition="center">
+    <GridForMoreResources>
+      <CardWithPopularLink
+        title="Utah Roads Data Model"
```
Comment (steveoh) [inline src/pages/products/sgid/transportation/road-centerlines.astro:121]: **polish**: lower case data model

**Review by steveoh — CHANGES_REQUESTED** [review]
looks pretty good other than the metadata file

**Comment by gregbunce** [comment]
> > a few of the blog links do not work b/c we're missing those blog pages.
> 
> every blog post is still published just not shown. Use the content/blog folder structure to see them. If they are not there then you deleted them and they will not work. If that is the case remove the link to it.

it's odd, i'm only seeing two total blog posts from 2024 (one road post). the road updates should have 3 alone.

File: src/data/downloadMetadata.ts (hunk 1 above)
Comment (gregbunce) [reply src/data/downloadMetadata.ts:2038]: another odd one, but the original itemid doesn't appear to work. maybe i'm missing something?

this works:
https://services1.arcgis.com/99lidPhWCzftIe9K/arcgis/rest/services/UtahRoads/FeatureServer/0

This does not:
https://services1.arcgis.com/478fbef62481427f95a3510a4707b24a/arcgis/rest/services/UtahRoads/FeatureServer/0

Comment (steveoh) [reply src/data/downloadMetadata.ts:2038]: Itemids are for arcgis hub. You are showing arcgis server urls. Totally separate.

File: src/data/downloadMetadata.ts [hunk 5]
```diff
@@ -2708,10 +2708,10 @@ export const dataPages: DownloadMetadata = {
     featureServiceHost: 'https://roads.udot.utah.gov/server/rest/services/Public/',
     mapServiceId: 'Freeway_Exits',
   },
-  'UDOT Routes ALRS': {
+  'Utah UDOT Routes ALRS': {
```
Comment (steveoh) [inline src/data/downloadMetadata.ts:2711]: If you make changes here, you should probably make the same change in the internal meta table.

**Comment by steveoh** [comment]
> it's odd, i'm only seeing two total blog posts from 2024 (one road post). the road updates should have 3 alone.

Is it possible those blog posts were written after we started the new website and weren't migrated?

File: src/pages/products/sgid/transportation/road-centerlines.astro [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+import GridForMoreResources from '@components/page/GridForMoreResources.astro';
+import GridForYouMightLike from '@components/page/GridForYouMightLike.astro';
+import Section from '@components/page/Section.astro';
+
+import { dataPages } from '@data/downloadMetadata';
+
+export const metadata: IMetadata = {
+  title: 'Utah Roads',
+  description: `Statewide road centerline dataset for Utah.`,
+  stewards: ['UGRC'],
```
Comment (steveoh) [inline src/pages/products/sgid/transportation/road-centerlines.astro:24]: Should this include udot and others?

**Review by steveoh — COMMENTED** [review]
I'll look into the blog posts on Monday

File: src/pages/products/sgid/transportation/road-centerlines.astro (hunk 6 above)
Comment (gregbunce) [reply src/pages/products/sgid/transportation/road-centerlines.astro:24]: yes, good catch.

File: src/data/downloadMetadata.ts (hunk 5 above)
Comment (gregbunce) [reply src/data/downloadMetadata.ts:2711]: true. i'm gonna revert this change, to not kick off a separate set of issues.

File: src/data/downloadMetadata.ts (hunk 1 above)
Comment (gregbunce) [reply src/data/downloadMetadata.ts]: gotcha.  i fixed it. thanks

**Comment by gregbunce** [comment]
steve: let me know if you need anything else from me. thanks!

**Pull request merged**